pkg/api/runtime_windows.go

```diff
@@ -0,0 +1,24 @@
+// +build windows
+
+package api
+
+import (
+	"runtime"
+	"syscall"
+
+	"zotregistry.dev/zot/pkg/log"
+)
+
+// DumpRuntimeParams dumps important runtime state such as file and socket limits.
+func DumpRuntimeParams(log log.Logger) {
+	evt := log.Info().Int("cpus", runtime.NumCPU()) //nolint: zerologlint
+
+	getmaxstdio := syscall.NewLazyDLL("kernel32.dll").NewProc("_getmaxstdio")
```

Comment:
This API doesn't report any useful limitation; the cap on open file handles is actually variable and based on available memory. Your best bet is to instead report the current open handle count.

Maybe something like:
```go
func getCurrentHandleCount() (uint32, error) {
	kernel32 := syscall.NewLazyDLL("kernel32.dll")

	getProcessHandleCount := kernel32.NewProc("GetProcessHandleCount")
	getCurrentProcess := kernel32.NewProc("GetCurrentProcess")
	currentProcess, _, _ := getCurrentProcess.Call()

	var handleCount uint32
	ret, _, err := getProcessHandleCount.Call(
		currentProcess,
		uintptr(unsafe.Pointer(&handleCount)),
	)

	if ret == 0 {
		return 0, err
	}
	
	return handleCount, nil
}

// ...

evt.Uint64("open handles", uint64(handleCount))
```

Reply:
(Or alternatively just not print anything on Windows since it's not very useful.)

Reply:
What we want is the max allowed file handles if possible, but if not this is the second best option.
Tested this PR as of now, and seems to work fine.